Gemfile

```diff
@@ -47,6 +47,8 @@ gem 'trestle'
 # For admin/user login security
 gem 'devise'
 
+# Added 02-15-19
+gem 'bootstrap', '~> 4.3.1'
```

Comment:
For our type of work it's usually good to leave a comment after the gem denoting what you're using it for. So for instance you could do `gem 'bootstrap', '~> 4.3.1' # CSS View framework`

Reply:
Sure, I left a descriptive comment on the line above the gem